RadixWallet/Features/AssetsFeature/Components/StakeUnitList/Components/StakeClaimNFTsView.swift

```diff
@@ -30,8 +48,17 @@ public struct StakeClaimNFTSView: View {
 		VStack(alignment: .leading, spacing: .small1) {
 			HStack {
```

Comment:
I think we should always use explicit spacing, and mostly 0 is best, with padding instead